src/main/asciidoc/index_en.adoc

```diff
@@ -145,9 +145,8 @@ image::ApacheIncubator.png[]
 [.twocolumns]
 --
 * Learn The Apache Way
-* Build the Community
+* Growing the community
```

Comment:
while i like teh word growing - it seem to imply that it's organic and just happens, building a community take work.

Reply:
Yeah, I agree, we need to provide an organic growing way for the podling, otherwise we need to always keeping an eye on it.